docs/configuration.md

```diff
@@ -284,6 +284,7 @@ the backend workers convert "Bytearray to utf-8 string" when the Content-Type of
 * `model_server_home` : Torchserve home directory. 
 * `max_request_size` : The maximum allowable request size that the Torchserve accepts, in bytes. Default: 6553500
 * `max_response_size` : The maximum allowable response size that the Torchserve sends, in bytes. Default: 6553500
+* `limit_max_image_pixels` : Default value is true. If this is set to "false", PIL.Image.MAX_IMAGE_PIXELS = None in backend default vision handler. 
```

Comment:
@lxning should we add some explanations that this setting would be helpful is running inference for images with size beyond 5MB (which I think is our current limit?).